changelog_unreleased/typescript/11515.md

```diff
@@ -1,6 +1,7 @@
-#### Avoid extra offset in arrow function body when using long types (#11515 by @kachkaev and @thorn0)
+#### [HIGHLIGHT]Avoid extra offset in arrow function body when using long types (#11515 by @kachkaev and @thorn0)
 
 Starting with Prettier 2.3.0, type declarations in arrow functions could affect function body offset. Changing the length of the type annotation could produce large diffs and thus increased the chance of git conflicts. To prevent this, function body offset was stabilized.
+However, this change can make a big difference from 2.4.1. Please be careful.
```

Comment:
The use of the term “offset” here wasn’t clear to me — maybe change it to “indentation?” Additionally, I would reword the last sentence like so:

```suggestion
**Note**: This change may affect a large number of lines in your codebase.
```
(or something similar?)